src/extract/query.rs

```diff
@@ -51,7 +51,7 @@ where
             .uri()
             .ok_or(UriAlreadyExtracted)?
             .query()
-            .ok_or(QueryStringMissing)?;
+            .unwrap_or_default();
```

Comment:
I don't think it is necessary to judge that the query string is empty, because `serde_urlencoded` will handle it. 🙂

Reply:
Yeah that makes sense.

I guess `extract::Form` has the same bug then. Wanna look at that as well? The fix should be identical.

We might be able to deprecate `QueryStringMissing` as well then since its not used anywhere else.

Reply:
I will check it out now.🙂